src/NServiceBus.Core/Sagas/Sagas.cs

```diff
@@ -33,13 +32,13 @@ internal Sagas()
         }
 
         /// <summary>
-        ///     See <see cref="Feature.Setup" />.
+        /// See <see cref="Feature.Setup" />.
         /// </summary>
         protected internal override void Setup(FeatureConfigurationContext context)
         {
             // Register the Saga related behaviors for incoming messages
-            context.Pipeline.Register(WellKnownStep.InvokeSaga, typeof(SagaPersistenceBehavior), "Invokes the saga logic");
-            context.Pipeline.Register("InvokeSagaNotFound", typeof(InvokeSagaNotFoundBehavior), "Invokes saga not found logic");
+            context.Pipeline.Register<PersistSagaBehavior.Registration>();
+            context.Pipeline.Register<LoadSagaBehavior.Registration>();
```

Comment:
@SzymonPobiega wasn't there a syntax to enforce order here? (not a blocker for this pull)